src/explore-education-statistics-admin/src/pages/release/data/components/ReorderIndicatorsList.tsx

```diff
@@ -0,0 +1,165 @@
+import ReorderList, {
+  FormattedIndicators,
+  ReorderProps,
+} from '@admin/pages/release/data/components/ReorderList';
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import tableBuilderService, {
+  Subject,
+  SubjectMeta,
+} from '@common/services/tableBuilderService';
+import orderBy from 'lodash/orderBy';
+import React, { useEffect, useState } from 'react';
+
+// EES-1243 - remove this when the backend is done
+const testSubjectMeta: SubjectMeta = {
+  filters: {},
+  indicators: {
+    Category1: {
+      id: 'category-1-id',
+      label: 'Category 1',
+      options: [
+        {
+          id: 'category-1-item-1-id',
+          value: 'category-1-item-1',
+          label: 'Category 1 Item 1',
+          unit: '',
+          name: 'category_1_item_1',
+        },
+        {
+          id: 'category-1-item-2-id',
+          value: 'category-1-item-2',
+          label: 'Category 1 Item 2',
+          unit: '',
+          name: 'category_1_item_2',
+        },
+        {
+          id: 'category-1-item-3-id',
+          value: 'category-1-item-3',
+          label: 'Category 1 Item 3',
+          unit: '',
+          name: 'category_1_item_3',
+        },
+      ],
+      order: 1,
+    },
+    Category2: {
+      id: 'category-2-id',
+      label: 'Category 2',
+      options: [
+        {
+          id: 'category-2-item-1-id',
+          value: 'category-2-item-1',
+          label: 'Category 2 Item 1',
+          unit: '',
+          name: 'category_2_item_1',
+        },
+      ],
+      order: 0,
+    },
+  },
+  locations: {},
+  timePeriod: {
+    hint: '',
+    legend: '',
+    options: [],
+  },
+};
+
+interface UpdatedIndicator {
+  id?: string;
+  indicatorItems: (string | undefined)[];
+}
+export type UpdateIndicatorsRequest = UpdatedIndicator[];
+
+interface Props {
+  subject: Subject;
+  onCancel: () => void;
+  onSave: (orderedIndicators: UpdateIndicatorsRequest) => void;
```

Comment:
Don't know if this should be made async, for when it's wired up to the real service call?

Reply:
I don't think it needs to be, I've tested it with the real call and the component populates correctly (just can't reorder it as not ids yet, so left the test data in for now)

Reply:
OK cool thanks for checking!